src/main/resources/jpedal-microservice.properties

```diff
@@ -8,4 +8,6 @@ libreOfficeTimeout=60000
 individualTTL=86400000
 fileDeletionService=false
 fileDeletionService.frequency=5
-maxConversionDuration=Infinity
\ No newline at end of file
+maxConversionDuration=Infinity
+remoteTracker.port=1099
+conversionMemoryLimit=3072
```

Comment:
In BuildVu/FormVu we have added some extra comments that I think would be nice to add for JPedal too - see
https://github.com/idrsolutions/buildvu-microservice-example/pull/71/files#diff-f16457c3fb10bb8c7bd0fa2b61c911fe755b7a2b2c97a1d0f7e37d8b7e5a0df0

We've also used 1024 for `conversionMemoryLimit` which is probably a more sensible default?